orderbook/src/main.rs

```diff
@@ -2,12 +2,15 @@ mod api;
 mod orderbook;
 
 use crate::orderbook::OrderBook;
-use std::sync::Arc;
+use std::{net::SocketAddr, sync::Arc};
 
 #[tokio::main]
 async fn main() {
+    tracing_subscriber::fmt::init();
     let orderbook = Arc::new(OrderBook::default());
     let filter = api::handle_all_routes(orderbook);
-    let result = warp::serve(filter).bind(([0, 0, 0, 0], 8080)).await;
-    println!("warp exited: {:?}", result);
+    let address = SocketAddr::new([0, 0, 0, 0].into(), 8080);
+    tracing::info!(%address, "serving order book");
+    warp::serve(filter).bind(address).await;
+    tracing::error!("warp excited");
```

Comment:
Is this excited or exited?
```suggestion
    tracing::error!("warp exited");
```

Reply:
Wait, is that how you spell the past tense of `to exit`? I thought it was `exited` and `excited` would be the adjective.